test/schema.js

```diff
@@ -28,12 +28,12 @@ const TestEnum = new GraphQLEnumType({
   values: {
     RED: { description: 'A rosy color' },
     GREEN: { description: 'The color of martians and slime' },
-    BLUE: { description: 'A feeling you might have if you can\'t use GraphQL' },
+    BLUE: { description: "A feeling you might have if you can't use GraphQL" },
```

Comment:
nit: `'` instead

Reply:
The project has lint-staged enabled, this is changing the single quote to a double quote each time I commit. Given all the changes in the diff seems that prettier has not been run on all the files yet. I think that it's better to open another PR where prettier is run on the entire project so that new PRs will have clean diffs.